parcel.go

```diff
@@ -14,47 +15,84 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("insert into parcel (status, client, address, created_at) values(?, ?, ?, ?)",
+		p.Status, p.Client, p.Address, p.CreatedAt)
+	if err != nil {
+		return 0, err
+		fmt.Errorf("Error during select request")
+	}
+
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	return int(id), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
-
+	row := s.db.QueryRow("select number, client, status, address, created_at from parcel where number = ?", number)
 	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
-
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
+	}
 	return p, nil
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// реализуйте чтение строк из таблицы parcel по заданному client
 	// здесь из таблицы может вернуться несколько строк
-
+	rows, err := s.db.Query("select number, client, status, address, created_at from parcel where client = ?", client)
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
-
+	if err != nil {
+		return res, err
```

Comment:
При ошибке лучше возвращать `nil`, чтобы не было "полуданных"
```suggestion
		return nil, err
```